src/cargo/ops/cargo_add/mod.rs

```diff
@@ -634,6 +688,42 @@ impl DependencyUI {
             })
             .collect();
     }
+
+    fn features(&self) -> (IndexSet<&str>, IndexSet<&str>) {
+        let mut activated: IndexSet<_> =
+            self.features.iter().flatten().map(|s| s.as_str()).collect();
+        if self.default_features().unwrap_or(true) {
+            activated.insert("default");
+        }
```

Comment:
Hi @Emilgardis! 
I'm trying to make a PR that touches this file and I wonder if this snippet is necessary, you add `"default"` to the activated features list and ignore it down below, can I remove it?

Reply:
oh, sorry, it is dead code indeed https://github.com/rust-lang/cargo/pull/11098#discussion_r974655861